core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -195,6 +197,23 @@ public static RaftJournalSystem create(RaftJournalConfiguration conf) {
     return system;
   }
 
+  private RaftJournalConfiguration processRaftConfiguration(RaftJournalConfiguration conf) {
+    // Override election/heartbeat timeouts for single master cluster
+    // if election timeout is not set explicitly.
+    // This is to speed up single master cluster boot-up.
+    if (conf.getClusterAddresses().size() == 1
+        && !ServerConfiguration.isSet(PropertyKey.MASTER_EMBEDDED_JOURNAL_ELECTION_TIMEOUT)) {
+      LOG.debug("Overriding election timeout to {} for single master cluster.",
+          SINGLE_MASTER_ELECTION_TIMEOUT_MS);
+      conf.setElectionTimeoutMs(SINGLE_MASTER_ELECTION_TIMEOUT_MS);
+      // Use the highest heartbeat internal relative to election timeout.
+      conf.setHeartbeatIntervalMs(Math.max(1, (SINGLE_MASTER_ELECTION_TIMEOUT_MS / 2) - 1));
```

Comment:
We don't need this max anymore right?

Reply:
Right. don't need to guard against incorrectly supplied values.